config/webpacker.yml

```diff
@@ -79,7 +79,8 @@ test:
 
   # CI precompiles packs prior to running the tests.
   # Also avoids race conditions in parallel_tests.
-  compile: false
+  compile: true
```

Comment:
I would not change the `compile` option here, we already unconditionally compile the assets in an earlier CI step.